CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [2.17.1] - 2024-06-12
 
+## Added
```

Comment:
Need to go in a new section above the `## [2.17.1]` with a `## [UNRELEASED]` title.

Reply:
Thank you for your correction. I have made the necessary changes as per your suggestion.